### PR TITLE
Createed ClinicDB from ERD

### PR DESCRIPTION
**Pull request by baqar-abbas**
### In this Project Milestone -: 

- we have created a Database from Entity Relationship diagram.
- Created a file named schema_based_on_diagram.sql where we implemented the database from the given diagram.
- Created relationships between tables by creating join table
- Created one to one and one to many relationships between respective tables.
- Created index on primary key of patients table.
- Created indexes on Foreign keys.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/* Create table invoice items cceated one to many relation with invoices and treatments*/
+
+CREATE TABLE invoice_items (
+    id serial primary key,
+    unit_price decimal(10,2),
+    quantity int,
+    total_price decimal(10,2),
+    invoice_id int REFERENCES invoices (id),
+    treatment_id int REFERENCES treatments (id)
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql:60]: - [ ] Please create indexes for the following foreign keys, as per the requirements.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    name varchar(100)
+);
+/* establish the many-to-many relationship between medical_histories and treatments*/
+
+CREATE TABLE medical_history_treatments (
+    id serial primary key,
+    medical_history_id int,
+    treatment_id int,
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id),
+    FOREIGN KEY (treatment_id) REFERENCES treatments (id)
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql:37]: - [ ] Please create indexes for the following foreign keys, as per the requirements.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @baqar-abbas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (baqar-abbas) [reply schema_based_on_diagram.sql:60]: Suggested changes by the code reviewer have been implemented

Comment (baqar-abbas) [reply schema_based_on_diagram.sql:60]: Screen shot attached below for your kind reference.
image

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (baqar-abbas) [reply schema_based_on_diagram.sql:37]: Suggested changes by the code reviewer has been implemented. Screen shot attached below for your reference.
image

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @baqar-abbas 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ All FOREIGN KEYS have indexes created

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by baqar-abbas** [comment]
Thanks

**Pull request merged**